lib/ansible/module_utils/redfish_utils.py

```diff
@@ -683,17 +703,26 @@ def set_one_time_boot_device(self, bootdevice):
         else:
             payload = {"Boot": {"BootSourceOverrideTarget": bootdevice}}
 
-        response = self.patch_request(self.root_uri + self.systems_uri, payload, HEADERS)
+        response = self.patch_request(self.root_uri + systems_uri, payload, HEADERS)
         if response['ret'] is False:
             return response
         return {'ret': True}
 
-    def set_bios_attributes(self, attr):
+    def set_multi_time_boot_device(self, bootdevice):
```

Comment:
This "multi" method isn't needed in this PR since this is for a "set" method called from redfish_command.

Reply:
Done